src/main/java/org/alliancegenome/curation_api/services/validation/dto/fms/HTPExpressionDatasetAnnotationFmsDTOValidator.java

```diff
@@ -117,18 +118,33 @@
 		}
 
 		if (CollectionUtils.isNotEmpty(dto.getCategoryTags())) {
-			List<VocabularyTerm> categoryTags = new ArrayList<>();
+			Map<String, VocabularyTerm> categoryTags = new HashMap<>();
 			for (String categoryTag : dto.getCategoryTags()) {
 				if (StringUtils.isNotEmpty(categoryTag)) {
-					VocabularyTerm tag = vocabularyTermService.getTermInVocabulary(VocabularyConstants.HTP_DATASET_CATEGORY_TAGS, categoryTag).getEntity();
-					if (tag == null) {
+					Map<String, Object> params = new HashMap<>();
+					params.put("name", categoryTag);
+					params.put("query_operator", "or");
+					params.put("synonyms", categoryTag);
+					SearchResponse<VocabularyTerm> searchResponse = vocabularyTermService.findByParams(new Pagination(), params);
+					boolean added = false;
+					if (searchResponse.getTotalResults() > 0) {
+						for(VocabularyTerm tag : searchResponse.getResults()) {
+							if(tag.getVocabulary().getName().equals("Data Set Category Tags") && (tag.getName().equals(categoryTag) || tag.getSynonyms().contains(categoryTag))) {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' is not followed by whitespace.
<!-- __reviewdog__:ChAzY2M3MGI1MDU3NWVmZmUzEgpjaGVja3N0eWxl -->